documentation/source/building.rst

```diff
@@ -62,6 +62,17 @@ Make Variables
 
       Passed to both the compile and execute phases of simulators with two rules, or passed to the single compile and run command for simulators which don't have a distinct compilation stage.
 
+.. make:var:: PLUSARGS
+
+      "Plusargs" are options that are starting with a plus (``+``) sign. 
+      They are passed to the simulator and are also available within cocotb as :attr:`cocotb.plusargs`.
```

Comment:
Perhaps should be the following, but maybe it works either way
```suggestion
      They are passed to the simulator and are also available within cocotb as :data:`cocotb.plusargs`.
```

Reply:
Done.